core/src/main/scala/besom/internal/logging.scala

```diff
@@ -353,6 +354,28 @@ object logging:
       .replace()
   }
 
+  /** For tests only
```

Comment:
please ad a hint on the intended usage to eliminate the guess work, the scribe docs are mostly not existent

Reply:
fixed